NUTemplateParser/NUParser.js

```diff
@@ -87,7 +87,7 @@ const parseString = (() => {
 
     // This regular expression detects instances of the
     // template parameter syntax such as {{foo}}, {{foo:someDefault}} or {{foo:['someDefault']}}.
```

Comment:
Would the change in regex potentially change what is described here ?

Reply:
Changes done in comment description.